app/src/main/res/layout/subscription_item.xml

```diff
@@ -37,12 +42,22 @@
         android:id="@+id/triangleCountView"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_alignParentRight="true"
-        android:layout_alignParentEnd="true"
         android:layout_alignParentTop="true"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
         app:backgroundColor="?attr/colorSecondary"
         app:corner="rightTop"
         app:primaryText="Test"
         app:primaryTextColor="?attr/colorOnSecondary"
         app:primaryTextSize="12sp" />
+
+    <CheckBox
+        android:id="@+id/selectCheckBox"
+        android:layout_width="32dp"
+        android:layout_height="32dp"
+        android:layout_centerVertical="false"
+        android:drawableLeft="@drawable/grey_border"
+        android:drawablePadding="0dp"
+        android:background="#44FFFFFF"
```

Comment:
To be honest, I don't really like that the selected item gets more transparent. It now looks like multiple transparent things overlayed on top of each other.
The checkbox background should be round like in the mock-up by keunes and be a bit more "strong". You could do that using a layout that is wrapped around the checkbox and has a VectorDrawable with a partial circle as its background.

Reply:
> I don't really like that the selected item gets more transparent

This doesn't bother me so much. But no strong opinion on this one.

> The checkbox background should be round like in the mock-up by keunes and be a bit more "strong".

I was also thinking the white should be less transparent. (And all in favour of more round background, of course. If not my mock-up, then I'd say at least a solid corner radius.)